HighPerformanceComputing.md

```diff
@@ -79,6 +79,10 @@ for later R releases.
     supported. Parallel map-reduce calls via the future framework are provided by packages
     `r pkg("future.apply")` for parallel versions of base-R apply functions, and
     `r pkg("furrr")` for parallel versions of purrr fuctions.
+-   The `r pkg("mirai")` package is a minimalist, highly-scalable framework for asynchronous
+    evaluation, allowing concurrent code execution locally or across the network, implementing
+    futures which automatically resolve upon completion. Uses the lightweight NNG C library as
+    backend through the high-performance `r pkg("nanonext")` binding.
```

Comment:
Please use full sentence.

Please try to shorten it as well.  Most entries try to get by with two lines.  

Please move it to the end of the section as section grow chronologically.

Reply:
Have amended the PR accordingly. Thanks.